recipes/tiledb-py/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: tiledb-py
+  version: {{ version }}
+
+source:
+  url: https://github.com/TileDB-Inc/TileDB-Py/archive/{{ version }}.tar.gz
+  sha256: 8223e468f6c01bd82be4fdfee2a3b58af205b6ddd0f812fc4000b983304c8f66
+
+build:
+  number: 0
+  skip: true  # [win or py2k]
+  script: python setup.py install --tiledb=$PREFIX --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - numpy 1.8.*  # [not (win and (py35 or py36))]
+    - tiledb 1.2.*
+
+  run:
+    - python
+    - numpy >=1.8  # [not (win and (py35 or py36))]
+    - tiledb 1.2.*
+
+test:
+  imports:
+    - tiledb
+  commands:
+    - python -c "import tiledb; tiledb.libtiledb.version()"
+
+about:
+  home: http://tiledb.io
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Python interface to the TileDB sparse and dense multi-dimensional array storage manager'
+  description: |
+    TileDB-Py is the python interface to the TielDB array storage manager.  
+    TileDB  is an efficient multi-dimensional array management system which introduces 
+    a novel on-disk format that can store dense and sparse array data with 
+    support for fast reads and updates and reads. It features excellent compression, and an efficient 
+    parallel I/O system with high scalability.
+  doc_url: https://api-reference.tiledb.io/python-api.html
+  dev_url: https://github.com/TileDB-Inc/TileDB-Py
+
+extra:
+  recipe-maintainers:
+    - stavrospapadopoulos
+    - jakebolewski
+    - tdenniston
```

Comment:
@tdenniston are you okay with being a maintainer?

Reply:
Yes.